content/blog/2024/grid-masonry-nuance.md

```diff
@@ -0,0 +1,495 @@
+---
+title: Choosing a Masonry Syntax in CSS
+sub: What makes something a 'grid', and what's at stake?
+author: miriam
+date: 2024-09-26
+image:
+  src: blog/2024/masonry.jpg
+  alt: >
+    A gallery of numbered images
+    in four columns
+tags:
```

Comment:
do we want to add `Grids`?

The plural there is weird to me, but that is what we've used for other posts.